agito.py

```diff
@@ -607,6 +607,37 @@ def append_branch_info(path, entry, message):
 	     + ("Subversion-branch: %s\n" % str(path)) \
 	     + ("Subversion-revision: %i\n" % entry.revision.number)
 
+def rewrite_commit_refs(path, entry, message):
+	"""Rewrites Subversion commit references to Git commits.
+
+	Supports `r123` and ``[123]`` style commits
+
+	Args:
+	  path: Path of the branch within Subversion.
+	  entry: Log entry with the details of this commit.
+	  message: The commit message text.
+	Returns:
+	  Altered commit message text with commit references replaced.
+	"""
+	def substitute(m):
+		ref = m.group(1)
+		commit = get_commit(path, ref)
+
+		if commit is None:
```

Comment:
What's the difference between the "commit is None" case here and the "else" case below? If there is no difference, just "if not commit:" will do.